regolith/tools.py

```diff
@@ -755,3 +755,34 @@ def update_schemas(default_schema, user_schema):
             updated_schema[key] = user_schema[key]
 
     return updated_schema
+
+
+def group(db, by):
```

Comment:
I am not sure what this is doing.  Maybe we can discuss.

Reply:
I will add an example to show what the group is doing.

Reply:
done.